sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -531,7 +531,7 @@ EventHubAsyncClient buildAsyncClient() {
         }
 
         if (scheduler == null) {
-            scheduler = Schedulers.elastic();
+            scheduler = Schedulers.boundedElastic();
```

Comment:
The java doc might need changing.
 <li>If no scheduler is specified, an {@link Schedulers#elastic() elastic scheduler} is used.</li>